upstairs/src/lib.rs

```diff
@@ -757,7 +781,9 @@ impl Work {
         for state in job.state.values() {
             match state {
                 IOState::New | IOState::InProgress => wc.active += 1,
-                IOState::Done | IOState::Skipped | IOState::Error => {
+                IOState::AckReady => wc.ack_ready += 1,
+                IOState::Error => wc.error += 1,
+                IOState::Acked | IOState::Done | IOState::Skipped => {
```

Comment:
`acked` is a field in WorkCounts, why not add to that?

Reply:
Yeah, I struggle with this method.
Its intent was to provide an easy summary of IO states that allow a caller to act on.
The reason I don't have a count for just acked is that the place that is looking for acked also wants to count the "Done" and "Skipped" IO states.